### PR TITLE
swaysettings: init at 2022-05-25

### PR DESCRIPTION
**Pull request by AndersonTorres**
###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/swaysettings/default.nix [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    libhandy
+    libxml2
+    xkeyboardconfig
+  ];
+
+  postPatch = ''
+    patchShebangs ./build-aux/meson/postinstall.py
+  '';
+
+  meta = with lib; {
```
Comment (bobby285271) [inline pkgs/applications/misc/swaysettings/default.nix:68]: Missing maintainers

File: pkgs/applications/misc/swaysettings/default.nix [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    gettext
+    meson
+    ninja
+    pkg-config
+    python3
+    vala
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
```
Comment (bobby285271) [inline pkgs/applications/misc/swaysettings/default.nix:50]: I think gsettings-desktop-schemas is required for `org.gnome.desktop.interface` schemas

https://github.com/ErikReider/SwaySettings/blob/c60a235061faf68ca1cf24d22b2aed73eb31e6c0/src/Pages/Themes/Themes_Page.vala#L5
https://nixos.org/manual/nixpkgs/unstable/#ssec-gnome-packaging

**Review by bobby285271 — COMMENTED** [review]
/cc @aacebedo (#175197)

**Comment by aacebedo** [comment]
How shall I correct the comment as it is not my PR anymore ?

File: pkgs/applications/misc/swaysettings/default.nix (hunk 2 above)
Comment (AndersonTorres) [reply pkgs/applications/misc/swaysettings/default.nix:51]: Good. I fixed that.

**Comment by AndersonTorres** [comment]
> How shall I correct the comments as it is not my PR anymore ?

What comments? O.o

**Comment by aacebedo** [comment]
I think I miss something in the PR process here. I made a PR here (https://github.com/NixOS/nixpkgs/pull/175197) for this app and I was of course expecting comments on it because it was my first PR.
Then this PR was opened (seemingly based on my original commit) and mine was closed.

Is this the normal process for PR in nixpkgs ?

**Comment by Pixinn** [comment]
> I think I miss something in the PR process here. I made a PR here (#175197) for this app and I was of course expecting comments on it because it was my first PR. Then this PR was opened (seemingly based on my original commit) and mine was closed.
> 
> Is this the normal process for PR in nixpkgs ?

Oh boy, how naive you are 🤣
He could have told you how to format your submission, but he obviously prefers to get all the credits for himself.

**Comment by AndersonTorres** [comment]
> Is this the normal process for PR in nixpkgs ?

Nope. I merely did the package, I noticed the other later.

**Comment by AndersonTorres** [comment]
> He could have told you how to format your submission, but he obviously prefers to get all the credits for himself.

Good idea. Novices need to suffer slowly.

**Pull request closed without merging**